packages/legend-lego/src/code-editor/CodeEditor.tsx

```diff
@@ -154,12 +165,31 @@ export const CodeEditor: React.FC<{
   );
 
   return (
-    <div
-      className={clsx('code-editor__container', {
-        'code-editor__container--padding': !hidePadding,
-      })}
-    >
-      <div className="code-editor__body" ref={textInputRef} />
-    </div>
+    <>
```

Comment:
this is not ideal, we should have child of `code-editor__container`, no need for `code-editor__background__container`, if you do this, it can throw off styling as the way we lay these `<div>s` out make them at the mercy of the containing div.